iron-multi-selectable.d.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-multi-selectable.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+/// <reference path="iron-selectable.d.ts" />
+
+declare namespace Polymer {
+
+  interface IronMultiSelectableBehavior extends Polymer.IronSelectableBehavior {
+
+    /**
+     * If true, multiple selections are allowed.
+     */
+    multi: boolean|null|undefined;
+
+    /**
+     * Gets or sets the selected elements. This is used instead of `selected` when `multi`
+     * is true.
+     */
+    selectedValues: any[]|null|undefined;
+
+    /**
+     * Returns an array of currently selected items.
+     */
+    readonly selectedItems: any[]|null|undefined;
+
+    /**
+     * Selects the given value. If the `multi` property is true, then the selected state of the
+     * `value` will be toggled; otherwise the `value` will be selected.
+     *
+     * @param value the value to select.
+     */
+    select(value: string|number): any;
+
+    /**
+     * UNUSED, FOR API COMPATIBILITY
+     */
+    _shouldUpdateSelection(): any;
```

Comment:
this is a getter

Reply:
Fixed by getter improvements from analyzer.